src/comment.rs

```diff
@@ -58,25 +58,23 @@ fn custom_opener(s: &str) -> &str {
 }
 
 impl<'a> CommentStyle<'a> {
-    /// Returns `true` if the commenting style covers a line only.
+    /// Returns `true` if the commenting style is a single line comment.
```

Comment:
Not to be a pedant but I think there's some important subtly that may be lost. Naturally it's possible to have a single-line, block style comment and my reading of the proposed changes loses that distinction

Reply:
I appreciate subtlety, so thanks for the comment.

Good point, that subtlety should be captured. The original wording also confused me in a similar fashion.

Two options:

1. ```    /// Returns `true` if the commenting style cannot span multiple lines.``` (added to PR as my choice)
2. Revert to as it is.

Happy to go either way.